.github/workflows/tests.yml

```diff
@@ -31,7 +31,9 @@ jobs:
       run: make test
 
     - name: Coveralls GitHub Action
+      if: matrix.python-version == '3.12'
       uses: coverallsapp/github-action@v2.3.6
       with:
         github-token: ${{ secrets.GITHUB_TOKEN }}
-        file: coverage.lcov
\ No newline at end of file
+        parallel: false
```

Comment:
Parallel builds need extra webhook to indicate finish, we don't need this complexity.